DeliveryCompany.AppForDrivers/Program.cs

```diff
@@ -96,7 +276,34 @@ private void GetWaybill(User user)
             }
         }
 
-        private User GetDriver()
+        private void UpdateWaybillMode(Package package)
+        {
+            if (package.ModeWaybill == ModeOfWaybill.Automatic)
```

Comment:
This seems strange. It works as I have checked, but with this condition, how it actually switched it from Auto to Manual mode? This seems to prevent it quite strongly.